src/components/LoansByCategory/HelpmeChoose/HelpmeChooseWrapper.vue

```diff
@@ -35,9 +40,18 @@ export default {
 			type: Boolean,
 			default: false
 		},
+		userData: {
+			type: Object,
+			default: () => {}
+		}
 	},
 	components: {
 		LoanCardController
+	},
+	computed: {
+		welcomeTitle() {
+			return this.isVisitor ? 'Need help choosing?' : `Need help choosing, ${this.userData.firstName}?`;
```

Comment:
Since `userData` is optional, you probably want optional chaining on `this.userData?.firstName`